operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1905,6 +1905,9 @@ message KialiConfig {
 message BaseConfig {
   // For Helm2 use, adds the CRDs to templates.
   google.protobuf.BoolValue enableCRDTemplates = 1;
+
+  // Configures the validation webbhook url.
+  string validationURL = 2;
```

Comment:
Wrong place - this should be in mesh config, it is not operator specific.

Reply:
removed